BUILD.md

```diff
@@ -15,10 +15,12 @@ To install cuML from source, ensure the following dependencies are met:
 9. NCCL (>=2.4)
 10. UCX [optional] (>= 1.7) - enables point-to-point messaging in the cuML standard communicator. This is necessary for many multi-node multi-GPU cuML algorithms to function.
 
-It is recommended to use conda for environment/package management. If doing so, a convenience environment .yml file is located in `conda/environments/cuml_dec_cudax.y.yml` (replace x.y for your CUDA version). This file contains most of the dependencies mentioned above (notable exceptions are `gcc` and `zlib`). To use it, for example to create an environment named `cuml_dev` for CUDA 10.0 and Python 3.7, you can use the follow command:
+It is recommended to use conda for environment/package management. If doing so, a convenience environment .yml file is located in `conda/environments/cuml_dec_cudax.y.yml` (replace x.y for your CUDA version). This file contains most of the dependencies mentioned above (notable exceptions are `gcc` and `zlib`). To use it, for example to create an environment named `cuml_dev` for CUDA 10.2 and Python 3.7, you can use the follow command:
 
 ```
-conda env create -n cuml_dev python=3.7 --file=conda/environments/cuml_dev_cuda10.0.yml
+conda create -n cuml_dev python=3.7
+conda activate cuml_dev
+conda env update --file=conda/environments/cuml_dev_cuda10.2.yml
 ```
```

Comment:
Both  ways are valid to create the environment, so I think keeping the succing one liner `conda env create...` would be good

Reply:
Sorry, are you suggesting that we leave it as the following?
```
conda env create -n cuml_dev python=3.7 --file=conda/environments/cuml_dev_cuda10.2.yml
```
If so, then we'll still run into #2748, where the Python version in the environment is *not* 3.7 as one would expect from this command. Based on the [conda issue](https://github.com/conda/conda/issues/4756), it looks like the recommended workaround is a two-step install. That issue is very old though, so maybe there's some other better workaround?

Or did you maybe mean something more like this?
```
conda create -n cuml_dev python=3.7
conda env update -n cuml_dev --file=conda/environments/cuml_dev_cuda10.2.yml
```
If so, then yep, I 100% agree. No reason to activate the environment just to update it. I'll go ahead and push that change, but tell me if I'm misunderstanding your suggestion.

Reply:
Ah hadn't seen that issue, the 2 line solution at the end looks perfect then

Reply:
Great! Just pushed the update